docs/docs/38-authoring/04-volumes.md

```diff
@@ -35,6 +35,24 @@ volumes: {
 }
 ```
 
+## Volumes with subpaths
+
+When referencing volumes in containers, subpaths are supported. This feature allows you to have containers share a volume easily. You can utilize volume subpaths by adding a query parameter in the `dir`'s volume url like so.
+
+```acorn
+containers: {
+    "my-app": {
+        image: "web"
+        // ...
+        dirs: {
+            "/data": "volume://my-data?subpath=data"
+        }
+    }
+}
+```
+
+If you'd like to see an example of subpaths in action you can take a look at our [Getting Started](../37-getting-started.md) guide. For more information on the concept of subpaths in general, you can checkout the official [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) on them.
```

Comment:
Can we provide enough info here to prevent a user from having to go to K8s docs?

Reply:
Yep! Good call. Let me update this to be more descriptive on subpaths.